src/ophys_etl/modules/decrosstalk/qc_plotting/pairwise_plot.py

```diff
@@ -581,25 +672,62 @@ def plot_pair_of_rois(roi0: OphysROI,
                       roi=roi1,
                       roi_color=color1)
 
-    # plot traces
-    raw_axis = plt.Subplot(fig, grid[1:4, 7:30])
-    unmixed_axis = plt.Subplot(fig, grid[6:9, 7:30])
+    # Set params for controlling width of trace axes.
```

Comment:
Can you give context for why the numbers below were chosen? (up to `n_zoom_timesteps`.

Reply:
Done (though it's not very satisfying; this is the first thing I tried and Natalia said "looks good.")